First Milestone: Hello Microverse Pull Request

HussenYesufAlli opened the pull request:
In this project I:
add index.html file that includes a header
add style.css file that includes one class with styles for my header
update readme file according to the template
add license file that use the MIT license

husyesuf47 committed:
Add index.html

husyesuf47 committed:
Add style.css

HussenYesufAlli committed:
add logo

HussenYesufAlli committed:
Update README.md

HussenYesufAlli committed:
Create  LICENSE.md

HussenYesufAlli committed:
Create LICENSE

HussenYesufAlli committed:
delete the  LICENSE file 

duplicated file  LICENSE file with out extension

HussenYesufAlli committed:
Create LICENSE

MIT License Template

HussenYesufAlli committed:
Delete LICENSE.md

duplicated license file

HussenYesufAlli committed:
Rename LICENSE to LICENSE.md

MIT license template

HussenYesufAlli committed:
Update README.md

add MIT license link

HussenYesufAlli committed:
Update README.md

HussenYesufAlli committed:
Update README.md

HussenYesufAlli committed:
Update linters.yml

change the node-version: "18.x" to "12.x"

tufoinnkuo10 commented on README.md:
```diff
@@ -1 +1,122 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## :rocket: Live Demo <a name="live-demo"></a>
```
- [ ] Great job for adding a logo, deploying your project and using the correct readme template. please always ensure that all the main headers from this [README template](https://github.com/microverseinc/readme-template) are available. the only optional one is the faq. you are missing the
[💻 Getting Started](https://github.com/microverseinc/readme-template#getting-started)
 [🤝 Contributing](https://github.com/microverseinc/readme-template#contributing),
[⭐️ Show your support](https://github.com/microverseinc/readme-template#support),
[🙏 Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
having them with little necessary information under them will make your README more professional. nice work.

Abenezer-Tilahun replied:
Fixed.

tufoinnkuo10 reviewed (changes requested):
Hi @HussenYesufAlli 

great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ gitignore file well used


Required Changes ♻️
Check the comments under the review

![tufoin](https://media.giphy.com/media/a6O5pzQ8pAPTCoeagK/giphy.gif)

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

HussenYesufAlli committed:
Update README.md

Include the missed main headers from this README template
  - Show your support
  -  Contributing
  - Acknowledgments

Abenezer-Tilahun reviewed (approved):
Hi @HussenYesufAlli  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
